kitchen.recipes.yml

```diff
@@ -85,3 +85,10 @@ suites:
     verifier:
       controls:
         - clusterstatusmgtd_files_created
+  - name: cron
+    run_list:
+      - recipe[aws-parallelcluster-test::docker_mock]
```

Comment:
@francesco-giordano I had to add this to make the Dokken test for RHEL8 pass, but I am not sure it's the correct approach.